backend/dataall/core/permissions/services/tenant_policy_service.py

```diff
@@ -25,9 +18,130 @@ def is_tenant_admin(groups: [str]):
 
         return False
 
+    @staticmethod
+    def validate_admin_access(username, groups, action):
+        if not TenantPolicyValidationService.is_tenant_admin(groups):
+            raise exceptions.UnauthorizedOperation(
+                action=action,
+                message=f'User: {username} is not allowed to manage tenant permissions',
+            )
+
+    @staticmethod
+    def validate_find_tenant_policy(group_uri, tenant_name):
+        if not group_uri:
+            raise exceptions.RequiredParameter(param_name='group_uri')
+        if not tenant_name:
+            raise exceptions.RequiredParameter(param_name='tenant_name')
+
+    @staticmethod
+    def validate_attach_tenant_policy(group, permissions, tenant_name):
+        if not group:
+            raise exceptions.RequiredParameter(param_name='group')
+        if not permissions:
+            raise exceptions.RequiredParameter(param_name='permissions')
+        if not tenant_name:
+            raise exceptions.RequiredParameter(param_name='tenant_name')
+
+    @staticmethod
+    def validate_save_tenant_policy(group, tenant_name):
+        if not group:
+            raise exceptions.RequiredParameter(param_name='group')
+        if not tenant_name:
+            raise exceptions.RequiredParameter(param_name='tenant_name')
+
```

Comment:
We can merge `validate_add_permission_to_tenant_policy_params` and `validate_add_permissions_params` into the same function right?